sql-statements/sql-statement-start-transaction.md

```diff
@@ -34,7 +34,12 @@ Query OK, 0 rows affected (0.01 sec)
 
 ## MySQL compatibility
 
-This statement is understood to be fully compatible with MySQL. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
+This statement is understood to be partly compatible with MySQL.
+
+* `START TRANSACTION` is equivalent to MySQL’s `START TRANSACTION WITH CONSISTENT SNAPSHOT`, that is, after `START TRANSACTION`, a `SELECT` statement (not `SELECT FOR UPDATE`) is executed to read data from any table in InnoDB.
```

Comment:
This sentence refers to InnoDB in a way that's kind of confusing. Can you make it more clear how the behavior in TiDB is different from the behavior in MySQL?